### DISCUSSION TITLE
Error:Cannot find module '../build/output/log'

### DISCUSSION THREAD

**Discussion by maohui66661** [vercel/next.js#39252] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### npx --no-install next info
![image](https://user-images.githubusercontent.com/7902405/182369711-cd290e38-2705-426c-b428-d4cbb234f77a.png)

### How are you deploying your application? (if relevant)

node & next start

### Describe the Bug

when exec 'next start'，show error：

![image](https://user-images.githubusercontent.com/7902405/182370510-9749754b-95eb-4e55-ae28-1012618a2d77.png)

### Expected Behavior

Start service normally

### Link to reproduction

d

### package.json
{
  "name": "thresh",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start -p 30928",
    "lint": "next lint"
  },
  "dependencies": {
    "@mdx-js/react": "^2.1.1",
    "@next/mdx": "^12.1.6",
    "antd": "^4.20.5",
    "next": "12.1.6",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-markdown": "^8.0.3",
    "remark-toc": "^7.2.0"
  },
  "devDependencies": {
    "@mdx-js/loader": "^2.1.1",
    "@types/node": "17.0.33",
    "@types/react": "18.0.9",
    "@types/react-dom": "18.0.4",
    "eslint": "8.15.0",
    "eslint-config-next": "12.1.6",
    "typescript": "4.6.4"
  }
}

/** @type {import('next').NextConfig} */
const remarkToc = require("remark-toc");

const withMDX = require("@next/mdx")({
  extension: /\.(md|mdx)$/,
  options: {
    remarkPlugins: [remarkToc],
    // rehypePlugins: [],
    // If you use `MDXProvider`, uncomment the following line.
    // providerImportSource: "@mdx-js/react",
  },
});

const nextConfig = {
  reactStrictMode: true,
};

### next.config.ts
module.exports = withMDX({
  reactStrictMode: true,
  pageExtensions: ["ts", "tsx", "js", "jsx", "md", "mdx"],
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    if (!isServer) {
      config.resolve.fallback = {
        fs: false,
      };
    }
    return config;
  },
});

**Comment by SwapnilpatilVineBrook** [comment]
I am getting same error, Is there any solution for this?

**Comment by la-flor** [comment]
Try deleting your node_modules directory and then reinstalling them.
